doc/life_of_a_language_feature.md

```diff
@@ -1,116 +1,141 @@
+under consideration. 
 
-All proposals should be linked from and link to the 'request' for
-the user problem/feature request they are trying to address.
+### Proposal Checklist
+
+- [ ] An issue labelled `feature` for discussion of a proposed solution.
+  - [ ] Links to and from the `request` issue they are trying to address.
+- [ ] A link to an initial feature specification writeup of the proposed
+solution in the `feature` issue.
+  - [ ] Located in a sub-directory with the name of the feature (`/
+[feature-name]/`), inside the
+[`working`](https://github.com/dart-lang/language/tree/master/working)
+directory of the language repository.
+  - [ ] File is named `feature-specification.md`.
+    - All written plans, specs, and materials must be written using [GitHub
+Markdown format](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) format, and must use the `.md` extension.
+  
+Proposals may be iterated on in-place.
 
 For smaller and non-controversial features, we will sometimes skip this step and
-proceed directly to the [Acceptance and
-implementation](#acceptance-and-implementation) phase.
+proceed directly to the [Acceptance](#acceptance) phase.
 
-### External (outside of the language team) feedback
+### External (outside of the language team) Feedback
 
-We expect to use the github issue tracker as the primary place for accepting
-feedback on proposals, solicited or unsolicited. If we solicit feedback, we
-anticipate opening an issue for discussion and feedback, but also encouraging
-filing and splitting off different issues for different threads of discussion.
+We use the Github issue tracker as the primary place for accepting feedback on
+proposals, solicited or unsolicited. 
 
-We generally expect to formally solicit at least one round of feedback on
-significant changes.
+If we solicit feedback, we will open an issue for discussion and feedback. We
+highly recommend filing and splitting off different issues for different threads
+of discussion. We generally expect to formally solicit at least one round of
+feedback on significant changes.
 
-## Acceptance and implementation
+## Acceptance
 
 If consensus is reached on a specific proposal and we decide to accept it, a
 member of the language team will be chosen to shepherd the implementation.
+
 The implementation will be tracked via two artifacts:
 
-  - A feature specification document
+  - The feature specification document
+  - The feature project
 
-  - A 'implementation' issue.
+### Feature Specification
 
 The 'feature specification' is **a single canonical writeup of the language
-feature** which is to serve as the implementation reference. Feature
-specifications use Markdown format. The file name should be
-`feature-specification.md`, and the feature specification should be located
-inside the `accepted/future-releases/` folder.
+feature** which is to serve as the implementation reference. The feature
```

Comment:
"is to serve" -> "serves".

(I've heard that future tense can be difficult for some where English isn't their primary language." Present tense also tends to be a little more concrete. Future tense can read as ambiguously possibly in the subjunctive.)